api/src/main/java/com/github/streamshub/console/api/SchemasResource.java

```diff
@@ -0,0 +1,93 @@
+package com.github.streamshub.console.api;
+
+import java.util.Base64;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.NotFoundException;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.jboss.logging.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.streamshub.console.api.support.KafkaContext;
+import com.github.streamshub.console.api.support.serdes.ArtifactReferences;
+import com.github.streamshub.console.api.support.serdes.MultiformatSchemaParser;
+
+import io.apicurio.registry.resolver.DefaultSchemaResolver;
+import io.apicurio.registry.resolver.SchemaResolver;
+import io.apicurio.registry.rest.client.RegistryClient;
+
+@Path("/api/schemas/{schemaId}")
+@Tag(name = "Schema Registry Resources")
+public class SchemasResource {
+
+    @Inject
+    Logger logger;
+
+    @Inject
+    ObjectMapper objectMapper;
+
+    @Inject
+    Map<String, KafkaContext> kafkaContexts;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(responseCode = "200", ref = "Configurations", content = @Content())
+    @APIResponse(responseCode = "404", ref = "NotFound")
+    @APIResponse(responseCode = "500", ref = "ServerError")
+    @APIResponse(responseCode = "504", ref = "ServerTimeout")
+    public Response getSchemaContent(
+            @Parameter(description = "Schema identifier")
+            @PathParam("schemaId")
+            String schemaId) {
+
+        int clusterTerm = schemaId.indexOf('.');
+
+        if (clusterTerm < 0) {
+            throw new NotFoundException("No such schema");
+        }
+
+        String clusterId = new String(Base64.getUrlDecoder().decode(schemaId.substring(0, clusterTerm)));
```

Comment:
I'm used to see REST libraries perform the decode for you.  Are we double encoding it or something?